R/slice_derivation.R

```diff
@@ -11,11 +11,8 @@
 #'   A function that performs a specific derivation is expected. A derivation
 #'   adds variables or observations to a dataset. The first argument of a
 #'   derivation must expect a dataset and the derivation must return a dataset.
-#'   The function must provide the `dataset` argument and all arguments
-#'   specified in the `params()` objects passed to the `arg` argument.
```

Comment:
`, params = c("dataset")` in line 94 should be removed.

Reply:
🤦of course! thanks!